### PR TITLE
data:swapnul

### PR DESCRIPTION
**Pull request by swapnul**
## Changes proposed
Add  https://github.com/swapnul profile to  LinkFree

##Check List (Check all the applicable boxes)
- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

NA

## Note to reviewers

NA

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4677"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please move your file to the `data` directory.
Let us know if you need any help with that. 🙂

File: data/swapnul.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Swapnil__404",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/swapnul",
+      "icon": "FaTwitter"
```
Comment (Panquesito7) [inline data/swapnul.json:14]: You might want to use the GitHub icon here.
```suggestion
      "icon": "FaGithub"
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by priyankarpal — APPROVED** [review]
Looks great 🎉

**Review by Sindhuinti — APPROVED** [review]
LGTM 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**